jobs/types/base_validation_errors.py

```diff
@@ -133,6 +133,16 @@ def __init__(self, model):
             'post_commit_status is %s' % model.post_commit_status)
 
 
+class InvalidPublicCommitStatusError(BaseAuditError):
+    """Error class for commit models with inconsistent public status values."""
+
+    def __init__(self, model):
+        super(InvalidPublicCommitStatusError, self).__init__(model)
+        self.message = (
+            'post_commit_status="%s" but post_commit_community_owned=%r' % (
```

Comment:
Why `%r` is used hereat the end of the string?

Reply:
Done